src/compiler/python_generator.cc

```diff
@@ -66,6 +66,9 @@ using std::vector;
 
 namespace grpc_python_generator {
 
+GeneratorConfiguration::GeneratorConfiguration()
```

Comment:
At this point, GeneratorConfiguration is just a constant.  Any chance we can just make these constants?